### PR TITLE
BuildTools Tasks of Marian Zlatev - Maven

### PR DESCRIPTION
**Pull request by NewestUser**
Project containing multiple modules.
Project can be compiled and packaged successfully with dependencies.

### CODE REVIEW

File: myproject/myproject-bl/pom.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      </plugin>
+    </plugins>
+  </build>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
```
Comment (mgenov) [inline myproject/myproject-bl/pom.xml]: why you are using junit 3.8.1 ?

**Comment by mgenov** [comment]
LGTM ! You can merge after you update to junit 4.

**Comment by mgenov** [comment]
Actually for the most of the projects in clouWay jarjar maven plugin is used: http://sonatype.github.io/jarjar-maven-plugin/ so it would be useful for all guys to see how it is working.

File: myproject/myproject-bl/pom.xml (hunk 1 above)
Comment (NewestUser) [inline myproject/myproject-bl/pom.xml]: Updated outdated dependency to 4.11

**Pull request merged**